pdc_management/pdc_management/doctype/pdc_management_settings/pdc_management_settings.json

```diff
@@ -0,0 +1,55 @@
+{
+ "actions": [],
+ "allow_rename": 1,
+ "creation": "2025-05-30 14:32:22.979470",
+ "doctype": "DocType",
+ "engine": "InnoDB",
+ "field_order": [
+  "maturity_notification_before",
+  "maturity_notification_template",
+  "notification_role"
+ ],
+ "fields": [
+  {
+   "fieldname": "maturity_notification_before",
+   "fieldtype": "Int",
+   "label": "Maturity Notification Before"
+  },
```

Comment:
Give proper description for the field. Whether it is in Hours, Days, Week, Month etc..